examples/integrated_gradients_transformers.ipynb

```diff
@@ -52,40 +55,117 @@
+    "    Tokenize the text sentences.\n",
     "    \n",
-    "    reviews = [REPLACE_NO_SPACE.sub(\"\", line.lower()) for line in reviews]\n",
-    "    reviews = [REPLACE_WITH_SPACE.sub(\" \", line) for line in reviews]\n",
+    "    Parameters\n",
+    "    ----------\n",
+    "    sentence:\n",
+    "        Sentence to be processed.\n",
+    "    tokenizer:\n",
+    "        Tokenizer to be used.\n",
     "    \n",
-    "    return reviews\n",
-    "\n",
-    "def process_sentences(sentence, \n",
-    "                      tokenizer, \n",
-    "                      max_len):\n",
-    "    \"\"\"Tokenize the text sentences.\n",
+    "    Returns\n",
+    "    -------\n",
+    "        Tokenized representation containing:\n",
+    "         - input_ids\n",
+    "         - attention_mask\n",
     "    \"\"\"\n",
+    "    # since we are using the model for classification, we need to include special char (i.e, '[CLS]', ''[SEP]')\n",
+    "    # check the example here: https://huggingface.co/transformers/v4.4.2/quicktour.html\n",
     "    z = tokenizer(sentence, \n",
-    "                  add_special_tokens = False, \n",
-    "                  padding = 'max_length', \n",
-    "                  max_length = max_len, \n",
-    "                  truncation = True,\n",
-    "                  return_token_type_ids=True, \n",
+    "                  add_special_tokens=True, \n",
+    "                  padding='max_length', \n",
+    "                  max_length=max_len, \n",
+    "                  truncation=True,\n",
     "                  return_attention_mask = True,  \n",
-    "                  return_tensors = 'np')\n",
+    "                  return_tensors='np')\n",
     "    return z"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def  hlstr(string: str , color: str = 'white') -> str:\n",
+    "    \"\"\"\n",
+    "    Return HTML markup highlighting text with the desired color.\n",
+    "    \"\"\"\n",
+    "    return f\"<mark style=background-color:{color}>{string} </mark>\"\n",
+    "\n",
+    "\n",
+    "def colorize(attrs: np.ndarray, cmap: str = 'PiYG') -> List:\n",
+    "    \"\"\"\n",
+    "    Compute hex colors based on the attributions for a single instance.\n",
+    "    Uses a diverging colorscale by default and normalizes and scales\n",
+    "    the colormap so that colors are consistent with the attributions.\n",
+    "    \n",
+    "    Parameters\n",
+    "    ----------\n",
+    "    attrs:\n",
+    "        Attributions to be visualized.\n",
+    "    cmap:\n",
+    "        Matplotlib cmap type.\n",
+    "    \"\"\"\n",
+    "    cmap_bound = np.abs(attrs).max()\n",
+    "    norm = mpl.colors.Normalize(vmin=-cmap_bound, vmax=cmap_bound)\n",
+    "    cmap = mpl.cm.get_cmap(cmap)\n",
+    "    return list(map(lambda x: mpl.colors.rgb2hex(cmap(norm(x))), attrs))\n",
+    "\n",
+    "\n",
+    "def display(X: np.ndarray, \n",
```

Comment:
Would it be possible to cut off when the actual sentence ends and [PAD] tokens are encountered?